src/command-testing.js

```diff
@@ -30,24 +35,18 @@ window.addEventListener('keyup', (event) => {
             preview.remove();
 
             // CTRL TAB HELD
-            chrome.runtime.sendMessage(null,
-                                numSwitches % 5, 
-                                (response)=>{
-                console.log("Sent key value: " + response);
-            });         
+            chrome.runtime.sendMessage(null, {contentPresses: numSwitches % 5});         
```

Comment:
Can we just send numswitches as a raw integer and then do the modulo by the queue size when we receive the message (in `switch-tab`)? I suppose for previews we will need to query the queue but for now it might help make it a little less weird when it always rolls over at 4.

Reply:
Sure that's definitely what we need to do, but I don't think this is within the scope of this PR. There are some other upcoming PRs that are meant to handle this.